central/reports/common/report_formatter.go

```diff
@@ -82,7 +84,7 @@ type WatchedImagesResult struct {
 
 // Format takes in the results of vuln report query, converts to CSV and returns zipped CSV data and
 // a flag if the report is empty or not
-func Format(deployedImagesResults []DeployedImagesResult, watchedImagesResults []WatchedImagesResult) (*bytes.Buffer, bool, error) {
+func Format(deployedImagesResults []DeployedImagesResult, watchedImagesResults []WatchedImagesResult, configName string) (*bytes.Buffer, bool, error) {
```

Comment:
Add a note to the function's comment that v1 configs should send configName as empty.